h/groups/views.py

```diff
@@ -15,13 +15,15 @@
 
 @view_config(route_name='group_create',
              request_method='GET',
-             renderer='h:groups/templates/create.html.jinja2',
-             permission="authenticated")
+             renderer='h:groups/templates/create.html.jinja2')
 def create_form(request):
     """Render the form for creating a new group."""
     if not request.feature('groups'):
         raise exc.HTTPNotFound()
 
+    if request.authenticated_userid is None:
+        raise exc.HTTPNotFound()
```

Comment:
Ok. I think I kind of agree with @robertknight that it's more pure in a Pyramid sense to always let the authz system handle authz so that in the case of an authz error the view never even gets called (and the solution he suggests sounds right to me). It means for example that we never have to test any auth stuff when writing tests for view callables. But I'm also fine with doing it the way you have it, I don't think it's a big problem.